example/android/build.gradle

```diff
@@ -1,12 +1,12 @@
 buildscript {
-    ext.kotlin_version = '1.3.50'
+    ext.kotlin_version = '1.6.10'
     repositories {
         google()
         jcenter()
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.5.0'
+        classpath 'com.android.tools.build:gradle:4.0.0'
```

Comment:
Did it actually work? In my environment, version 4.1.0 is required in Flutter v3.3.

```groovy
classpath 'com.android.tools.build:gradle:4.1.0'
```

Reply:
@Kurogoma4D  I have Updated Code
Kindly Review & merge the PR.